test/e2e/specs/bank-separate.spec.ts

```diff
@@ -3,7 +3,7 @@ describe('bank-separate', () => {
         await page.goto(`${location}/bank-separate`, {
             waitUntil: 'networkidle2'
         });
-        await page.waitFor(1000);
+        await page.waitFor(2000);
```

Comment:
These `waitFor` with a set value seems like a code spell.

You have a `waitUntil` in the goto above? And a magic number `waitFor`, You also have a waitFor in a lot of the it blocks that run below adding even more timeout with a magic number. 

This seems like a flaw in the tests.